themes/default/views/editor/objects/screen_html.php

```diff
@@ -82,4 +82,56 @@
 
 	<div class="editorBottomPadding"><!-- empty --></div>
 	
-	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
\ No newline at end of file
+	<?php print caSetupEditorScreenOverlays($this->request, $t_object, $va_bundle_list); ?>
+	<script type="text/javascript">
+		(function(initSplitSuffix) {
+
+			initSplitSuffix(window.jQuery);
+
+		}(function($) {
+
+				$(function () {
+					var reg = /^\s*\.?\s?(\d*)\s*(.*)\s*$/,
```

Comment:
Since this is only used once (line 97), just inline it.  Then you don't need to worry about thinking of a better variable name ;-)